packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/toolbar.scss

```diff
@@ -18,14 +18,17 @@
             & .dc-btn__icon {
                 padding-right: 0.4rem;
             }
+
```

Comment:
@prince-deriv do we need extra line breaks in this file?🙂

Reply:
removed line breaks ✅